sources/packages/web/src/composables/useRules.ts

```diff
@@ -11,14 +11,14 @@ export function useRules() {
     return "SIN is required.";
   };
 
-  const checkNotesLengthRule = (notes: string) => {
+  const checkNotesLengthRule = (notes: string, noteTitle?: string) => {
     if (notes) {
       return (
         checkMaxCharacters(notes, NOTES_MAX_CHARACTERS) ||
         `Max ${NOTES_MAX_CHARACTERS} characters.`
       );
     }
-    return "Note body is required.";
+    return `${noteTitle ? noteTitle : "Note body"} is required.`;
```

Comment:
I am not sure where the content `Note body` came from but since we have a validator specific for "Notes" I would expect all of them to provide the same message unless there is more than one on the same form.
Can we just assume that "Note is required." will be the default message?
@hellolynn-tbtb FYIA

Reply:
That's ok to me to do the change. If I change the validation I'll need to change the other "Notes" titles:
![image](https://user-images.githubusercontent.com/78114138/223863599-31fc51cf-faa9-4f14-8201-f8a691c1dbb4.png)

Reply:
I did not follow this, why can't we just leave the validation message as "Note is required."?

Reply:
Because in the case that I showed the field title is "Note body" and the validation is going to show "Note is required".

Reply:
I'm happy to go with "Note" and "Note is required" @andrepestana-aot